components/gitpod-db/go/json.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License.AGPL.txt in the project root for license information.
+
+package db
+
+import (
+	"encoding/json"
+	"fmt"
+	"gorm.io/datatypes"
+)
+
+type EncryptedJSON[T any] datatypes.JSON
```

Comment:
This is made generic, to better communicate what are the contents of the encrypted data